src/views/questionView.js

```diff
@@ -26,3 +26,23 @@ export const createQuestionElement = (questionNumber, question) => {
 
   return element;
 };
+
+export const initScore = () => {
+  const score = document.body.appendChild(document.createElement('div'));
+  score.id = "score"
+  score.textContent = "Score: 0"
+}
+
+let count = 0;
+
+export const updateScore = (correct, selected) => {
+
+  const showScore = () => {
+    const score = document.getElementById('score')
```

Comment:
Here we should also avoid accessing the DOM from the views directory. We can pass the htmlelement in as a parameter to this function instead.